docs/source/simulators/CircuitSimulator.rst

```diff
@@ -1,5 +1,30 @@
-CircuitSimulator
-================
+Circuit Simulator
+=================
+
+The *Circuit Simulator* is the "default" simulator in DDSIM.
+It can handle all input supplied as :code:`QuantumComputation` from the QFR. This includes, but is not limited to,
+OpenQASM and real files as well as pre-defined algorithm like QFT, Grover, and random clifford circuits.
+
+Capabilities:
+
+- Exact Simulation :cite:p:`zulehner2019advanced,DBLP:conf/iccad/ZulehnerHW19`
+- Weak Simulation :cite:p:`DBLP:conf/dac/HillmichMW20`
+- Approximate Simulation with different strategies :cite:p:`10.1145/3530776,DBLP:conf/date/HillmichKMW21`
+
+    - Fidelity-driven
+    - Memory-driven
+
+
+Usage in Python
+###############
+
+Available backends in the provider :code:`DDSIMProvider`:
+
+- :code:`qasm_simulator` to simulate and sample from the resulting state
+- :code:`statevector_simulator` to simulate and explicitly return the state vector (which will require an exponential amount of memory in the number of qubits)
```

Comment:
Just because I stumbled upon it: It probably wouldn't hurt to also describe the non-Qiskit usage of these Simulators.
So people can decide whether they want to go the Qiskit way or just roll with DDSIM alone.

Reply:
I'll add a small example :)